neural_compressor/torch/quantization/quantize.py

```diff
@@ -185,7 +206,12 @@ def convert(
     if is_ipex_available and (
         isinstance(quant_config, StaticQuantConfig) or isinstance(quant_config, SmoothQuantConfig)
     ):
-        model_info = quant_config.get_model_info(q_model, example_inputs)
+        if isinstance(quant_config, SmoothQuantConfig):
+            model_info = quant_config.get_model_info(
+                q_model, quant_config.alpha, quant_config.act_algo, example_inputs, inplace=True
```

Comment:
why not keeping the same interface and use self.alpha. inplace is always True as we have checked it at the begining.